Public/Src/Engine/Scheduler/Scheduler.cs

```diff
@@ -4857,10 +4907,17 @@ private void ProcessFileChanges(LoggingContext loggingContext, SchedulerState sc
             if (m_configuration.Schedule.InputChanges.IsValid)
             {
                 inputChangeList = InputChangeList.CreateFromFile(
-                    loggingContext, 
+                    loggingContext,
                     m_configuration.Schedule.InputChanges.ToString(Context.PathTable),
                     m_configuration.Layout.SourceDirectory.ToString(Context.PathTable),
                     DirectoryTranslator);
+
+                // Only master maintain the list
+                if (!IsDistributedWorker && "DynamicCodeCov".Equals(Environment.GetEnvironmentVariable("[Sdk.BuildXL]qCodeCoverageEnumType")))

```

Comment:
>"DynamicCodeCov".Equals(Environment.GetEnvironmentVariable("[Sdk.BuildXL]qCodeCoverageEnumType") [](start = 44, length = 96)

This is icky. Is it just temporary for testing? If no we need to figure out a better way to pass this configuration

Reply:
Should this just be based on any process in the build graph having ChangeAffectedInputListWrittenFilePath set?

---
In reply to: [317211659](https://github.com/microsoft/BuildXL/pull/748#discussion_r317211659) [](ancestors = 317211659)

Reply:
I will remove this unnecessary check. m_configuration.Schedule.InputChanges and [Sdk.BuildXL]qCodeCoverageEnumType" should present or absent together.